<img src="http://willfreyman.org/assets/img/onag_big1.jpg" width="350" />

# IB290: Topics in Phylogenetics
# UC Berkeley

* Grad Seminar 460
* Class Number 67082
* Fall, 2019
* Thursdays 10:10 - 11:00 AM (Berkeley time!!!)
* 1002 VLSB
* Class website: [https://mikeryanmay.github.io/IB290/](https://mikeryanmay.github.io/IB290/)

## Instructors

* Carl Rothfels -- crothfels@berkeley.edu
* John Huelsenbeck -- johnh@berkeley.edu
* Mike May -- mrmay@berkeley.edu

## Format

The instructors will lead a series of introductory sessions (lectures) for the first four or five meetings. Subsequent meetings will be student-led. Each student will be responsible for leading at least one session (potentially more, depending on enrollment and interest) on a topic of their choice, in consultation with the instructors. That student will be responsible for developing an annotated bibliography of the important papers on the topic, including both “classics” and the most recent works, for selecting two or so of the papers for the class to read, and for preparing a short (15min) presentation on the topic. The remainder of the session will be devoted to discussion.

## Course Texts

There are no formal texts required for this seminar, but participants are encouraged to consult *Tree Thinking: An Introduction to Phylogenetic Biology* (Baum & Smith 2012) and also *Inferring Phylogenies* (Felsenstein 2004) and *Molecular Evolution: A Statistical Approach* (Yang 2014).

## Useful Info

Here are some useful links on statistics and phylogenetic software: [https://mikeryanmay.github.io/IB290/useful](https://mikeryanmay.github.io/IB290/useful)

## Possible Topics for Student Led Discussions

Possible topics include (but are _not limited to_): modeling migration/hybridization/introgression (phylogenetic networks); state-dependent diversification models; fossil data (e.g. fossilized-birth-death process/tip dating vs node calibrations); modeling gene duplication and loss; polymorphism-aware (PoMo) models; covarion/hidden state models; model testing (e.g. Bayes factors, AIC); model adequacy (e.g. posterior predictive tests); demographic inference (e.g., inferring population size changes through time); the multispecies coalescent, genetrees/species tree; the BAMM controversy; divergence time dating; inferring selection from sequence alignments; alignment inference (alignment-phylogeny co-inference?); community phylogenetics; spatial phylogenetics; inferring phylogenies from morphological data; heterogeneous models, e.g., non-stationary base frequencies; ancestral state reconstruction; phylogenomics (strengths, pitfalls?); inferring correlated evolution of traits; assessing support; hypothesis testing of relationships; reconstructing morphological evolution on a phylogeny; ABC approaches; hidden Markov models; mixture models; species delimitation; microbial community analysis; quartet-based methods (e.g. SVDquartets); etc.

Please sign up for a topic at [this google sheet](https://docs.google.com/spreadsheets/d/1NO0to-26p4oEMG6dpgEWpOWcJRHIVJgSL_brp_LjLeI/edit?usp=sharing). *Please consult an instructor before settling on a topic.* When you have chosen a topic and schedule a week to lead, update this README by following these instructions: [updating the course website](https://mikeryanmay.github.io/IB290/git_workflow).

See the [2017 course website](https://wf8.github.io/IB290/) or the [github repository](https://mikeryanmay.github.io/IB290/2017_files) for examples of reading lists and annotated bibliographies.

## Schedule:

Our first meeting will be on Thursday, September 5.

### September 5: Introductions

*Who are we, and want to do we want to get out of this seminar?*
* Outline of the course
* Outline of “phylogenetics”, i.e., introduction to trees—what is a phylogeny?—and tree thinking—what does this tree say?
* Discussion of topics to select for future class sessions
* Introduction to Git
* Reading: chapters 1, 2, and 3 of Tree Thinking
* Slides: [introduction to git](https://mikeryanmay.github.io/IB290/lecture_slides/git.pdf)

<!-- * Slides: [Rothfels Intro Slides](https://wf8.github.io/IB290/lecture_slides/Mtg01_intro.ppt) -->

### September 12: Introduction to probability theory

*Introduction to probability, estimation, and inference:*
* Introduction to probability and inference (on the whiteboard).

### September 19: Introduction to tree inference and Markov models of character change

*Introduction to the application of Markov models to tree inference (Part 1):*
* What is a Markov model?
* Properties and uses of Markov models
* Pruning algorithm
* Slides: [Likelihood-based phylogenetic inference](https://mikeryanmay.github.io/IB290/lecture_slides/likelihood.pdf)

### September 26: Introduction to tree inference and Markov models of character change

*Introduction to the application of Markov models to tree inference (Part 2):*
* More on Markov models
* More about substitution models
* Slides: [Likelihood-based phylogenetic inference](https://mikeryanmay.github.io/IB290/lecture_slides/likelihood.pdf)


### October 3: Introduction to tree inference and Markov models of character change

No power!

### October 10:  Introduction to tree inference and Markov models of character change
* More on Markov models
* More about substitution models
* Slides: [Likelihood-based phylogenetic inference](https://mikeryanmay.github.io/IB290/lecture_slides/likelihood.pdf)


### October 17: TBD

* Students:
* Reading:
* Annotated bibliography:

### October 24: TBD

* Students: Jenna & Sonia
* Reading:
	1. Nakhleh, L. "Computational approaches to species phylogeny inference and gene tree reconciliation." Trends in Ecology and Evolution, 28:112 (2013). (https://www.sciencedirect.com/science/article/pii/S0169534713002139) [LINK to PDFs](https://github.com/mikeryanmay/IB290/tree/master/readings/Oct24/)
	2. Liu, Y. et al. "Resolution of the ordinal phylogeny of mosses using targeted exons from organellar and nuclear genomes." Nature Communications 20:2485 (2019). (https://www.nature.com/articles/s41467-019-09454-w) [LINK to PDFs](https://github.com/mikeryanmay/IB290/tree/master/readings/Oct24/)
* Annotated bibliography:


### October 31: A spooky TBD

* Students:
* Reading:
	1. De Queiroz, K. 2007. Species concepts and species delimitation. Systematic Biology 56: 879–886. [LINK to PDF](https://github.com/mikeryanmay/IB290/tree/master/readings/Oct31/)
	2. Yang, Z., and B. Rannala. 2014. Unguided Species Delimitation Using DNA Sequence Data from Multiple Loci. Molecular Biology and Evolution 31: 3125–3135. [LINK to PDF](https://github.com/mikeryanmay/IB290/tree/master/readings/Oct31/)
	3. Leache, A.D., T. Zhu, B. Rannala, and Z. Yang. 2019. The Spectre of Too Many Species. Systematic Biology 68: 168–181. [LINK to PDF](https://github.com/mikeryanmay/IB290/tree/master/readings/Oct31/)
* Annotated bibliography: [LINK](https://github.com/mikeryanmay/IB290/tree/master/bibliographies/speciestreeinference_msc.pdf)

### November 7: TBD

* Students: Jackie
* Reading:
	1. Schluter, D., Price, T., Mooers, A. Ø., & Ludwig, D. (1997). Likelihood of ancestor states in adaptive radiation. Evolution, 51(6), 1699-1711. [LINK](https://github.com/mikeryanmay/IB290/tree/master/readings/Nov7/)
	2. Joy, J. B., Liang, R. H., McCloskey, R. M., Nguyen, T., & Poon, A. F. (2016). Ancestral reconstruction. PLoS computational biology, 12(7), e1004763. [LINK](https://github.com/mikeryanmay/IB290/tree/master/readings/Nov7/)
	3. Olsen, A. M. (2015). Exceptional avian herbivores: multiple transitions toward herbivory in the bird order Anseriformes and its correlation with body mass. Ecology and Evolution, 5(21), 5016-5032. [LINK](https://github.com/mikeryanmay/IB290/tree/master/readings/Nov7/)
* Annotated bibliography: [LINK](https://github.com/mikeryanmay/IB290/tree/master/bibliographies/Ancestralstatereconstruction.pdf)

### November 14: TBD

* Students: Jenn and Ben
* Reading:
	1. Quental, T. B., & Marshall, C. R. (2010). Diversity dynamics: molecular phylogenies need the fossil record. Trends in Ecology & Evolution, 25(8), 434-441. [LINK](https://github.com/mikeryanmay/IB290/tree/master/readings/Nov14/)
	2. Heath, T. A., Huelsenbeck, J. P., & Stadler, T. (2014). The fossilized birth–death process for coherent calibration of divergence-time estimates. Proceedings of the National Academy of Sciences, 111(29), E2957-E2966. [LINK](https://github.com/mikeryanmay/IB290/tree/master/readings/Nov14/)
	3. O’Reilly, J. E., Dos Reis, M., & Donoghue, P. C. (2015). Dating tips for divergence-time estimation. Trends in Genetics, 31(11), 637-650. [LINK](https://github.com/mikeryanmay/IB290/tree/master/readings/Nov14/)
* Annotated bibliography: [LINK](https://github.com/mikeryanmay/IB290/tree/master/bibliographies/Age_FBD-Wagner_Muddiman.pdf)

### November 21: Fossils in Biogeographic Inference

* Students: Reilly and Jaemin
* Reading:
	1. Ree, R.H., and Smith, S.A., 2008, Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis: Systematic Biology, v. 57, p. 4-14. doi: 10.1080/10635150701883881. [LINK](https://github.com/mikeryanmay/IB290/blob/master/readings/Nov21/Ree_and_Smith_2008_ML-inference-of-geographic-range-evolution.pdf)
	2. Matzke, N.J., 2013, Chapter Four: Incorporation of fossils in likelihood analyses of historical biogeography using models for imperfect detection, in Probabilistic historical biogeography: new models for founder-event speciation, imperfect detection, and fossils allow improved accuracy and model testing [Ph.D. thesis]: Berkeley, University of California, 240 p. [LINK](https://github.com/mikeryanmay/IB290/blob/master/readings/Nov21/Matzke_2013_Probabilistic-Historical-Biogeography.pdf)
	3. Silvestro, B., Zizka, A., Bacon, C.D., Cascales-Miñana, B., Salamin, N., and Antonelli, A., 2016, Fossil biogeography: a new model to infer dispersal, extinction, and sampling from paleontological data: Phil. Trans. R. Soc. B 371, 20150225. doi: 10.1098/rstb.2015.0225. [LINK](https://github.com/mikeryanmay/IB290/blob/master/readings/Nov21/Silvestro_et_al_2016_Fossil_biogeography.pdf)
* Annotated bibliography: [LINK](https://github.com/mikeryanmay/IB290/tree/master/bibliographies/fossil_biogeography_annotated_bibliography.pdf)


### December 5: Allopolyploid phylogenetics

* Students: Shirley Zhang, Keir Wefferling and Ixchel Gonzalez-Ramirez
* Readings:
	1. Jones, G. R. (2017). Bayesian phylogenetic analysis for diploid and allotetraploid species networks. BioRxiv, 129361 
	2. Oxelman, B., Brysting, A. K., Jones, G. R., Marcussen, T., Oberprieler, C., & Pfeil, B. E. (2017). Phylogenetics of allopolyploids. Annual Review of Ecology, Evolution, and Systematics, 48, 543–557 
	3.Wen, D., Yu, Y., & Nakhleh, L. (2016). Bayesian inference of reticulate phylogenies under the multispecies network coalescent. PLoS genetics, 12(5), e1006006. 

* Annotated bibliography:

### December 12: TBD

* Students:
* Reading:
* Annotated bibliography:
